src/main/java/com/diamondfire/helpbot/bot/command/impl/other/mod/MuteCommand.java

```diff
@@ -114,4 +116,17 @@ public void run(CommandEvent event) {
         });
         
     }
+    
+    private MessageEmbed getMuteEmbed(Member moderator, String reason, long duration) {
+        EmbedBuilder muteEmbedBuilder = new EmbedBuilder()
+                .setTitle("You have been muted!")
+                .setDescription(
+                        "**Reason**: " + reason + "\n" +
+                        "**Moderator**: " + moderator + "\n" +
```

Comment:
Yea just please don't include this. This info can be given by admins if needed, but isn't needed for the normal player.